ai-baseline/qwen-baseline/main.py

```
@@ -0,0 +1,94 @@
+)
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+
+answer = """
+Business Process Re-engineering is a crucial activity to conduct while implemeting an EHR. It can promote performance by making sure that all interrelated processes, activities, and businesses are smoothly aligned. Thus, enhancing overall quality levels by reducing errors and their huge costs. It can also make sure that all stakeholders are involved, aware and familiar with the coming changes given that EHR implementation can be absulotley overwhelming to hospital staff as observed in litrature. Having the staff familiar does not only affect their stress levels resulting from the change, but also causes them to collaborate and navigate their way through the change better leading to a faster and less problematic organizational shift to the newly implemented EHR.
+
+"""
+messages = [
+    {
+        "role": "system",
+        "content": """
+You are a supportive and encouraging computer science professor grading a student\’s short-answer quiz. Use the following question and rubric to assign a numerical score out of 16 and provide a short explanation. Be VERY lenient in your grading — the goal is to reward ANY signs of understanding, even if incomplete or partially incorrect. Give VERY generous partial points. Even if the answer is not perfect, reward the student for any effort.
```

Ruff (0.14.3)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)